examples/nginx-grafana-metabase/README.md

```diff
@@ -0,0 +1,79 @@
+# Visualizing Nginx Access Logs using Grafana and Metabase
+In my last [blog post](https://www.timeplus.com/post/analyzing-nginx-access-logs), I showed how Timeplus Proton can be used to analyze web traffic in production using just the log files generated by an Nginx web server. The article explored real-time analysis and historical analysis but the analysis wasn't visualized. 
+
+This article will remedy the lack of rich visualizations by showing how to do so using open source tools like [Grafana](https://grafana.com/) and [Metabase](https://www.metabase.com/).
+
+
+## Prerequisities
+In addition to Timeplus Proton, the previous article used the following technologies: 
+* [Amazon Virtual Private Cloud](https://aws.amazon.com/vpc/) (VPC);
+* [Amazon Elastic Compute Cloud](https://aws.amazon.com/ec2/) (EC2);
+* [NFS](https://en.wikipedia.org/wiki/Network_File_System);
+* [IPinfo](https://ipinfo.io/);
+* [Ghost](https://ghost.org/);
+* [Nginx](https://nginx.org/en/).
+  
+The first two require access to an AWS account. The last two products need to have been running in production, in order to have meaningful access log data to experiment with. 
+
+This article does away with most of those prerequisities making it easier to get started. It builds on the same analysis of the previous article but you only need: 
```

Comment:
prerequisities is a tpo. 

For "This article does away with most of those prerequisities", I don't quite get it. Do you need you still need those prerequisites to run the examples in the blog, or doesn't need it. Maybe change a way to express it.

Reply:
Thanks for catching the typo.

I've reworded a bit. Hopefully it's clearer.